### ISSUE TITLE
cjose_jwe_import fails for valid JWE compact serialization

### ISSUE THREAD

**Issue by sashei** [cisco/cjose#59]
Steps to reproduce:
call `cjose_jwe_import` with a JWE compact serialization that does not include the encrypted key portion (ie, `[something]..[something].[something].[something]`)

expected behavior: `cjose` creates a `_cjose_jwe_int` with an empty `_cjose_jwe_part_int` in the second position of the JWE part array
actual behavior: `cjose` returns `null` and gives a `CJOSE_ERR_INVALID_ARG` error.

notes:
alg is `ECDH-ES`
enc is `A256GCM`

**Comment by vladikoff** [comment]
( cc @rfk to keep in the loop )

**Comment by linuxwolf** [comment]
@sashei  do you have a sample JWE output?

My guess is ECDH-ES -- which I see didn't actually get implemented yet )-:

**Comment by vladikoff** [comment]
@linuxwolf 

```
eyJhbGciOiJFQ0RILUVTIiwia2lkIjoiMDhFVWxIbEFGT1NqcHNtbmd6Zm5vU0pLNjg3VUlCV3RadzJsb0l5QVdSMCIsImVwayI6eyJrdHkiOiJFQyIsImNydiI6IlAtMjU2IiwieCI6IlkxN1JtWklMVzV5LUtXREpRVzR1cy11Y1FLdGVXdUd4T0xMakxnR3Y1cU0iLCJ5IjoiaFc3QkQ4QjRMU1JILUZEd0RjUEZGUnpXX1VLMWg5bDR1M3ZNWlJvZldXQSJ9LCJlbmMiOiJBMjU2R0NNIn0..VOmkw7kwO59R1ORP.ym0hGSnu5jamJ-2XKMZ19bwVrE5JFrk9P8hEUbJj2npH4Z7FOoZsHnAcl4TQiQNW_Yvfffu391RzJCZKkgvbs9wjWIUL4B0NLvAjigPxVdTbgdmkXsDdIY4XnNcpQFWpEkzKQldJJKvcSeulYZCSbajK4U_ob-WO0CPCaNseGY46eHZoPlvSSgAi-VtVkziYfkYr-lwgpFELEMacwGzVWwMfws5q92jEiN8bKN9v6Kmr7n-tKFN2uc4QvMEmLA-5KkULI0oVFxn5bQASWwwUPgfMVaduRx28NwYwLYMSzAA.I47uOk6dfmtdqZAJXJrEtg	
```

![image](https://user-images.githubusercontent.com/128755/32869682-54a3aad6-ca46-11e7-9735-2843f547d0b8.png)

**Comment by linuxwolf** [comment]
@vladikoff thanks, that confirms it's ECDH-ES .. which is missing.

**Comment by linuxwolf** [comment]
related to #60

**Comment by linuxwolf** [comment]
Closing as this has been addressed in 1250eff10fa178937aea1924887d114c8ba943c2.
